app/templates/core-data/admin-unit/index.hbs

```diff
@@ -17,6 +17,22 @@
               <:label>Naam</:label>
               <:content>{{@model.adminUnit.name}}</:content>
             </Item>
+               <Item>
+                  <:label>Naam</:label>
```

Comment:
I see 2 naam labels, this should be different I think

Reply:
This was my mistake but will push it